README.md

```diff
@@ -49,6 +49,11 @@ type Service interface {
     Do(context.Context, Request) (Response, error)
 }
 
+type Request struct {
+    // (...)
```

Comment:
What about this comment? Is this struct missing something?

Reply:
Is to indicate that in addition to what already exists in the struct has the trace.Trace parameter